mattlianje/gregarious-wolf
# Pitch Solo Extraction

- Created sendPitch.py , gets only pitch from youtube URL
- Also edited import-clip code to run locally easliy (when testing)
- ALSO added progress prompts to main.py (if we ever even use it)

**TESTING**
We want to test locally, so for a random video (ideally kind of short just for testing) do the following. DJ Trump Video: "_https://www.youtube.com/watch?v=JZRXESV3R74_"

- Turn on localhost and everything
- Create "_**labelled**_" table locally (you might have to create the columns too idk)
- Add a "pitch" column to **_"labelled"_** table
- cd `import-clips`
- `npm start https://www.youtube.com/watch?v=JZRXESV3R74 log`
This should fill the local table with empty rows for all time intervals.

- cd `audio-analysis`
- `python sendPitch.py log`
- Enter the DJ Link above (put whatever for tag)
When finished running, the "pitch" column for that URL in **_"labelled"_** table should be filled out.

Might have dependencies and stuff.

NEXT STEPS: Run this code on our prod DB.

![image](https://user-images.githubusercontent.com/26189114/75752888-95e96580-5cf7-11ea-9e38-0ec8979dcd10.png)
